src/Plugin/Visualisation/Style/AxisChart.php

```diff
@@ -128,6 +128,7 @@ public function defaultConfiguration() {
    */
   public function settingsForm(array $form, FormStateInterface $form_state) {
     $form = parent::settingsForm($form, $form_state);
+    $dvf_helpers = \Drupal::service('dvf.helpers');
```

Comment:
@rob-duplock dependency injection.

Reply:
All instances of `\Drupal::service('dvf.helpers')` have been replaced with dependency injection